adding info about iCub CPUs (#126)
icub-tech-iit/documentation: docs/icub_operating_systems/index.md

## Robot OS

> There are two versions of the customized OS that runs on the machine inside iCub head*:

Reviewer: ```suggestion
There are two versions of the customized OS that run on the machine inside iCub head.

!!! note
    For further details, please refer to the [iCub CPU boards](../icub_cpu_boards/icub_cpu_boards.md)
```

| COMEXPRESS KONTRON   |   | COMe-cKL6i7-7600 8GB32S | Kontron        | COM-Express Type6, Kontron COMe-cKL6i7-7600 8GB RAM onboard and 32GB SLC eMMC 5.0 (Core i7-7600U 1.7GHz) | COM-Express Type6   | iCub 2.5-2.6 and 2.7  | Core i7-7600U             | 8+8GB RAM | onboard 32GB        |                                                                           |
|                  |   |                         |                |                                                                                                          |                     |                           |                           |           |                     |                                                                           |

<center> <img src ="../img/atom.png" width=300>       
</center>

ATOM board 

<center> <img src ="../img/congatec.png" width=300>       
</center>

CONGATEC

<center> <img src ="../img/kontron.png" width=300>       
</center>

> KONTRON

Reviewer: Also, you may want to use the following format:

```
<center>
  <figure>
    <img src="../img/filename"/>
    <figcaption><b>Figure Caption</b></figcaption>
  </figure>
</center>
```

Reviewer: Don't forget to push the image files as well.

- [_CUDA workstation_](./other-machines/cuda-workstation.md) - a GPU-equipped machine with CUDA software installed
- [_The iCub Server_](./other-machines/icub-server-from-scratch.md) - a machine acting only as server (for Network gateway, NTP, NFS, DHCP and DNS services)
- [_The iCub Laptop Server_](./other-machines/icub-server-laptop.md) - a single machine, usually a laptop, acting both as server (for Network Gateway, ntp and NFS serviceas) as well as robot console

> 
> 
> * for details please refer to [iCub CPU boards](./icub_cpu_boards/icub_cpu_boards.md)

Reviewer: ```suggestion

```

- [_CUDA workstation_](./other-machines/cuda-workstation.md) - a GPU-equipped machine with CUDA software installed
- [_The iCub Server_](./other-machines/icub-server-from-scratch.md) - a machine acting only as server (for Network gateway, NTP, NFS, DHCP and DNS services)
- [_The iCub Laptop Server_](./other-machines/icub-server-laptop.md) - a single machine, usually a laptop, acting both as server (for Network Gateway, ntp and NFS serviceas) as well as robot console

> 
> 
> * for details please refer to [iCub CPU boards](./icub_cpu_boards/icub_cpu_boards.md)

Reviewer: ```suggestion

```
